contract/readme/DESCRIPTION.rst

```diff
@@ -2,3 +2,4 @@ This module enables contracts management with recurring
 invoicing functions. Also you can print and send by email contract report.
 
 It works for customer contract and supplier contracts.
+Contracts show in portal.
```

Comment:
You need to leave an empty line for proper paragraph syntax, and also proper grammar:
```suggestion

Contracts are shown in portal.
```

You can extend more the explanation, adding in CONFIGURE.rst who are those that see the contracts in portal, and optionally in USAGE.rst some screenshots and steps.